.github/workflows/android.yml

```diff
@@ -0,0 +1,21 @@
+name: Android CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
```

Comment:
What about we build on Windows and macOS too like:

```
jobs:
  build:yaml
    runs-on: ${{ matrix.os }}
    strategy:
      matrix:
        os: [macos-latest, ubuntu-latest, windows-latest]
```

Reply:
@bruno-garcia sounds good, I just used the default template and see if it works and what else we'd need to do to make it better/complete.